NEMSpy/nems.py

```diff
@@ -2,26 +2,81 @@
 from enum import Enum
 from textwrap import indent
 
-from NEMSpy.model import Model, ModelType
+from .model import Model, ModelType
+from .model.ocean import OceanModel
+from .model.atmospheric import AtmosphericModel
+from .model.wave import WaveModel
+from .model.hydrological import HydrologicalModel
 
 
 class ModelRelationMethod(Enum):
     REDISTRIBUTE = 'redist'
 
 
+class ModelConnector:
+    def __init__(
+            self, source: ModelType, destination: ModelType,
+            method: ModelRelationMethod = ModelRelationMethod.REDISTRIBUTE):
+        self.source = source
+        self.destination = destination
+        self.method = method
+
+
 class ModelSequence:
-    def __init__(self, duration: timedelta = None, indent: str = '  '):
-        self.duration = duration if duration is not None else \
-            timedelta(hours=1)
-        self.indent = indent
-        self.__models = {model_type: None for model_type in ModelType}
-        self.relations = {model_type: {} for model_type in ModelType}
+    def __init__(self, duration: timedelta):
+        self.duration = duration
+        self.connectors = []
 
-    @property
-    def models(self) -> {ModelType: Model}:
-        return {model_type: model
-                for model_type, model in self.__models.items()
-                if model is not None}
+    def __str__(self) -> str:
+        lines = []
+        for conn in self.connectors:
+            lines += f'{conn.source} -> {conn.destination}    :remapMethod={conn.method}'
+        # TODO: Order of execution is not well-defined on the current API
+        for model_type in self.models:
+            lines += f'{model_type}'
```

Comment:
This is VERY important!! We need a method to specify the _order of execution_. This can probably be derived from the way the connectors are defined, but we need more clarification here, and probably some additional implementation.